app/models/grantee.rb

```diff
@@ -11,43 +11,56 @@ def hses_link
     links[:html]
   end
 
-  def centers_total
-    attributes[:numberOfCenters]
+  def centers
+    current_grant&.centers
   end
 
   def complaints_per_fy
-    attributes[:totalComplaintsFiscalYear]
+    grants.map(&:complaints_per_fiscal_year).each_with_object({}) do |count, obj|
```

Comment:
[suggestion] would you mind adding a quick comment here to explain what it's doing? I get from talking to you this morning that you're taking all of the grants associated with a grantee and combining their fiscal year info for one set of fiscal year values across grants, but it's a little confusing to read the code so even just a short comment might help!